src/components/courses/Courses.tsx

```diff
@@ -232,8 +233,8 @@ function UpdateCourse({ course, updateCourse, close }: { course: Course, updateC
                     </Dropdown.Button>}
                 </Card>
                 <br/>
-                <Card title={<><CalendarOutlined /> Kurstage: </>}>
-                    {!isEditMode && (course.subcourses ? course.subcourses[0].lectures.map(l => <Tag>{ new Date(l.start).toLocaleDateString() }</Tag>) : "")}
+                <Card title={<><CalendarOutlined /> Kurszeiten: </>}>
+                    {course.subcourses ? course.subcourses[0].lectures.map(l => <Tag>{ moment(l.start).format("DD.MM.YY hh:mm") }</Tag>) : ""}
```

Comment:
I think this should be `HH:mm` to have the common date format for Germany

Reply:
Fixed.